velox/functions/lib/RegistrationHelpers.h

```diff
@@ -61,7 +60,7 @@ void registerNonSimdizableScalar(const std::vector<std::string>& aliases) {
   registerFunction<T, TReturn, Varbinary, Varbinary>(aliases);
   registerFunction<T, TReturn, bool, bool>(aliases);
   registerFunction<T, TReturn, Timestamp, Timestamp>(aliases);
-  registerFunction<T, TReturn, Date, Date>(aliases);
+  registerFunction<T, TReturn, int32_t, int32_t>(aliases);
```

Comment:
This line should be removed entirely. int32_t should be a simdizable scalar I think,